### DISCUSSION TITLE
getting error while clicking on the download button Atom 1.57.0

### DISCUSSION THREAD

**Discussion by mary-celine** [atom/atom#22662] [Q&A]
Getting below error while clicking on the Download button 
This XML file does not appear to have any style information associated with it. The document tree is shown below.
<Error>
<Code>BlobNotFound</Code>
<Message>The specified blob does not exist. RequestId:23853ea7-e01e-003c-450b-6b0ba2000000 Time:2021-06-27T04:19:25.8717535Z</Message>
</Error>

**Comment by UziTech** [comment]
duplicate of #22650
